README.md

```diff
@@ -38,7 +38,7 @@ About conda-forge
 
 conda-forge is a community-led conda channel of installable packages.
 In order to provide high-quality builds, the process has been automated into the
-conda-forge GitHub organization. The conda-forge organization contains one repository
+conda-forge GitHub organization. The conda-forge organization contains one repository 
```

Comment:
Yeah, this is a little weird. Why is the URL being changed? I updated to the correct one recently. Should we merge this?